components/brave_rewards/resources/page/reducers/rewards_reducer.ts

```diff
@@ -98,7 +98,7 @@ const rewardsReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State
     }
     case types.ON_CLEAR_ALERT: {
       let ui = state.ui
-      if (!ui[action.payload.property]) {
+      if (ui[action.payload.property] === undefined) {
```

Comment:
Do we want to take this branch if `ui[action.payload.property] === null`? If so we can do either:

```js
ui[action.payload.property] === undefined || ui[action.payload.property] === null
```

or 

```js
ui[action.payload.property] == undefined
```

Reply:
we can't do null as null is valid value so should not be ignored